### PR TITLE
Custom Hostname update function added

### PR DESCRIPTION
**Pull request by pcaminog**
Custom Hostname Update function added
## Description

Function to be able to modify the custom hostname SSL settings

## Has your change been tested?

Test file updated and new library used to modify the custom hostnames

## Types of changes

What sort of change does your code introduce/modify?

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -51,6 +51,7 @@ golang.org/x/net v0.0.0-20191119073136-fc4aabc6c914/go.mod h1:z5CRVTTTmAJ677TzLL
 golang.org/x/net v0.0.0-20191125084936-ffdde1057850 h1:Vq85/r8R9IdcUHmZ0/nQlUg1v15rzvQ2sHdnZAj/x7s=
 golang.org/x/net v0.0.0-20191125084936-ffdde1057850/go.mod h1:z5CRVTTTmAJ677TzLLGU+0bjPO0LkuOLi4/5GtJWs/s=
 golang.org/x/net v0.0.0-20191126235420-ef20fe5d7933/go.mod h1:z5CRVTTTmAJ677TzLLGU+0bjPO0LkuOLi4/5GtJWs/s=
+golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553 h1:efeOvDhwQ29Dj3SdAV/MJf8oukgn+8D8WgaCaRMchF8=
```
Comment (jacobbednarz) [inline go.sum]: Why is this required for this change?

Comment (pcaminog) [reply go.sum]: I was testing and forgot to deleted it.
I've removed the line now

File: custom_hostname.go [hunk 2]
```diff
@@ -55,8 +55,19 @@ type CustomHostnameListResponse struct {
 // hostname in the given zone.
 //
 // API reference: https://api.cloudflare.com/#custom-hostname-for-a-zone-update-custom-hostname-configuration
-func (api *API) UpdateCustomHostnameSSL(zoneID string, customHostnameID string, ssl CustomHostnameSSL) (CustomHostname, error) {
-	return CustomHostname{}, errors.New("Not implemented")
+func (api *API) UpdateCustomHostnameSSL(zoneID string, customHostnameID string, ssl CustomHostnameSSL) (*CustomHostnameResponse, error) {
```
Comment (patryk) [inline custom_hostname.go]: Is there a reason you're changing return type?

Comment (pcaminog) [reply custom_hostname.go]: I just wanted to follow same style as it is in createCustomHostname, reponse from the API is similar on borh cases.

**Comment by patryk** [comment]
Thanks!
